sysutils/topgrade/Portfile

```diff
@@ -273,4 +273,4 @@ cargo.crates \
     ws2_32-sys 0.2.1 d59cefebd0c892fa2dd6de581e937301d8552cb44489cdff035c6187cb63fa5e \
     xattr 0.2.2 244c3741f4240ef46274860397c7c74e50eb23624996930e484c16679633a54c \
     xml-rs 0.6.1 e1945e12e16b951721d7976520b0832496ef79c31602c7a29d950de79ba74621 \
-    zip 0.5.6 58287c28d78507f5f91f2a4cf1e8310e2c76fd4c6932f93ac60fd1ceb402db7d
+    zip 0.5.6 58287c28d78507f5f91f2a4cf1e8310e2c76fd4c6932f93ac60fd1ceb402db7d
```

Comment:
I'll revert this one change, because we do want Portfiles (and all text files) to end with the required newline character.

Reply:
Thanks for fixing that, I'll try to remember this.